pkg/zsio/zsio.go

```diff
@@ -3,17 +3,16 @@ package zsio
 import (
 	"io"
 
-	"github.com/mccanne/zq/pkg/zsio/bzson"
-	"github.com/mccanne/zq/pkg/zsio/ndjson"
-	"github.com/mccanne/zq/pkg/zsio/table"
-	"github.com/mccanne/zq/pkg/zsio/text"
-	"github.com/mccanne/zq/pkg/zsio/zeek"
-	"github.com/mccanne/zq/pkg/zsio/zjson"
-	zsonio "github.com/mccanne/zq/pkg/zsio/zson"
 	"github.com/mccanne/zq/pkg/zson"
-	"github.com/mccanne/zq/pkg/zson/resolver"
 )
 
+type Flags struct {
+	Utf8       bool
```

Comment:
Nit: UTF8 would be a more conventional name.

Reply:
ok